uint8 id_agent
string status